modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -1,13 +1,18 @@
 [[clients-sleses]]
-= Registering {sleses} Clients
+= Registering {sll} Clients
 
-This section contains information about registering traditional and Salt clients running {sleses} ({es}) operating systems.
-{es} clients are based on {rhel} or {centos}.
-They are sometimes also called SLESES, RES or {redhat} {es}.
+This section contains information about registering traditional and Salt clients running {sll} operating systems.
+{sll} clients are based on {rhel} or {centos}.
 
-The {es} software channels provided by {suse} only provide updates to packages, they do not provide the packages themselves.
-To register {es} clients, you need to register the {es} Product (outlined below) to create the necessary base channel, then import any required {redhat} or {centos} packages as custom child channels.
-You must obtain the initial packages directly from {redhat} or {centos} before you can apply the updates provided by the {es} software channels.
+
+[NOTE]
+====
+{sll} clients are sometimes also called {sleses} (SLESES), Liberty, RES or {redhat} {es}.
+====
+
+The {sll} software channels provided by {suse} only provide updates to packages, they do not provide the packages themselves.
```

Comment:
Is that still true ? Now SLL is a "real" product, that can be installed. Is that to say we have no "pool" packages ?

Reply:
Yes, this is still true. Only the updates. SUSE don't provide any pool channel from scc when it comes to SLL.

Reply:
ok, thanks for clarification!